app/assets/javascripts/blacklight/blacklight.js

```diff
@@ -0,0 +1,12 @@
+// Customize which blacklight javascripts are imported
+// See: https://github.com/projectblacklight/blacklight/wiki/Using-Sprockets-to-compile-javascript-assets#customization-options
```

Comment:
Using sprockets rather than vite here, since the blacklight_range_limit js assumes that the blacklight js is in sprockets